.travis.yml

```diff
@@ -71,51 +71,53 @@ matrix:
 
   allow_failures:
     - python: 3.5
-      TESTS: py3status
+      env: TESTS=py3status
 
 # Dependencies installation commands
 install:
   - pip install -U pip
   # codecov is the interface to codecov.io; see after_success
   - pip install codecov
-  - "[ $TWISTED != trunk ] || pip install git+https://github.com/twisted/twisted"
-  - "[ $TWISTED = latest ] || [ $TWISTED = trunk ] || pip install Twisted==$TWISTED"
-  - "[ $SQLALCHEMY = latest ] || pip install sqlalchemy==$SQLALCHEMY"
+  - "if [ $TWISTED == trunk ]; then pip install git+https://github.com/twisted/twisted ; fi"
```

Comment:
single `=` is POSIX shell compliant, but `==` works here, because under the covers Travis is using bash which allows `==`.